mistral/mistral-7b-trt-llm/README.md

```diff
@@ -0,0 +1,70 @@
+[![Deploy to Baseten](https://user-images.githubusercontent.com/2389286/236301770-16f46d4f-4e23-4db5-9462-f578ec31e751.svg)](https://app.baseten.co/explore/mistral)
+
+# Mistral-7B Truss
+
+This is a [Truss](https://truss.baseten.co/) for Mistral 7B. This README will walk you through how to deploy this Truss on Baseten to get your own instance of Mistral 7B.
+
+**Warning: This example is only intended for usage on a single A100, changing your resource type for this deployment will result in unsupported behavior**
+
+## Truss
+
+Truss is an open-source model serving framework developed by Baseten. It allows you to develop and deploy machine learning models onto Baseten (and other platforms like [AWS](https://truss.baseten.co/deploy/aws) or [GCP](https://truss.baseten.co/deploy/gcp). Using Truss, you can develop a GPU model using [live-reload](https://baseten.co/blog/technical-deep-dive-truss-live-reload), package models and their associated code, create Docker containers and deploy on Baseten.
```

Comment:
AWS and GCP are broken links. Let's just drop them. We dropped the page for sagemaker from docs last week.